modules/axfr/axfr.go

```diff
@@ -163,6 +159,9 @@ func (s *GlobalLookupFactory) Initialize(c *zdns.GlobalConf) error {
 			return err
 		}
 	}
+	if c.IterativeResolution == true {
```

Comment:
Can we get more info about why this is true? I know it throws a segfault, can we paste into this PR (for future reference) a trace, etc?